### DISCUSSION TITLE
Achievement Badge Showcase feedback

### DISCUSSION THREAD

**Discussion by yasinnaal** [community/community#58315] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi, 

Great news to lunch the beta Achievement badge program. it will be very helpful and useful in maintaining users reputation, achievements, interaction... is there any guide for Currently Available Missions and Badges?

Please share with us.

Good luck,

Yasin

**Comment by Gustavonn07** [comment, marked as answer]
Hey how's it going? yes there is, you can see in these two repositories

1. How to get achieviments:
https://github.com/4xmen/Get-Github-Achievements

2. How to update yours achieviments and get them:
https://github.com/Schweinepriester/github-profile-achievements

**Comment by Gildaciolopes** [comment]
❤

**Comment by Gildaciolopes** [comment]
💖
